Sources/SwiftUITooltip/TooltipViewExtension.swift

```diff
@@ -8,27 +8,27 @@
 import SwiftUI
 
 public extension View {
-    func tooltip<TooltipContent: View>(@ViewBuilder content: @escaping () -> TooltipContent) -> some View {
+    func tooltip<TooltipContent: View>(isEnabled: Binding<Bool>, @ViewBuilder content: @escaping () -> TooltipContent) -> some View {
         let config: TooltipConfig = DefaultTooltipConfig.shared
 
-        return modifier(TooltipModifier(config: config, content: content))
+        return modifier(TooltipModifier(isEnabled: isEnabled, config: config, content: content))
```

Comment:
What if instead of enabling/disabling the tooltip in the `TooltipModifier` we do it here? Would make one less parameter for the tooltip code?

Reply:
I tried this suggestion but then I was unable to explicitly specify the `transition` for the overlay. It defaults to an opacity transition but I think it will be better if we can make transition style configurable through the config later.

This is what I tried:
```
@ViewBuilder
    func tooltip<TooltipContent: View>(enabled: Bool, @ViewBuilder content: @escaping () -> TooltipContent) -> some View {
        let config: TooltipConfig = DefaultTooltipConfig.shared

        if enabled {
            modifier(TooltipModifier(config: config, content: content))
        } else {
            self
        }
    }
```

Reply:
Pushed changes to include transition configuration. 
So now we can do something like this:
```
var tooltipConfig = DefaultTooltipConfig()
    init() {
        tooltipConfig.transition = .opacity.combined(with: .scale(scale: 2)).combined(with: .slide)
    }
```

https://user-images.githubusercontent.com/729622/164913428-309daca7-6801-486c-8c42-11bbd299b462.mp4